exporter/trace/stackdriver/stackdriver.go

```diff
@@ -28,13 +28,32 @@
 // 	trace.RegisterExporter(exporter)
 //
 // The package uses Application Default Credentials to authenticate.  See
-// https://developers.google.com/identity/protocols/application-default-credentials
+// https://developers.google.com/identity/protocols/application-default-credentials.
```

Comment:
Also could we not put the "." right after the URL? Some browsers such as Safari at times mistakenly pick up the "." and create invalid URLs.